### DISCUSSION TITLE
Unreal Engine 5 support

### DISCUSSION THREAD

**Discussion by DenisShockapp** [nxrighthere/UnrealCLR#243] [General]
So, they released UE5 Preview 1 !
And it looks like it broke UnrealCLR again.
Are there plans to release an update for this soon?

**Comment by nxrighthere** [comment]
I'll look into that.

**Comment by nxrighthere** [comment]
You can use https://github.com/nxrighthere/UnrealCLR/pull/248 provided by @tonetechnician, but I haven't checked it yet.

**Comment by tonetechnician** [comment]
@DenisShockapp I've been testing with this and so far so good. That being said, I haven't really played with the classes that changed.

**Comment by DenisShockapp** [comment]
@tonetechnician Thank you. It helped us a lot and we are using it now. However, there really are quite a few problems there, due to the use of double. So, it still requires an additional update for UE5. But at least it works.
